extensions/amp-render/1.0/test/test-amp-render.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../amp-render';
+import {toggleExperiment} from '../../../../src/experiments';
+import {waitFor} from '../../../../testing/test-helper';
+
+describes.realWin(
+  'amp-render-v1.0',
+  {
+    amp: {
+      extensions: ['amp-mustache:0.2', 'amp-bind:0.1', 'amp-render:1.0'],
+    },
+  },
+  (env) => {
+    let win, doc, element;
+
+    beforeEach(async function () {
+      win = env.win;
+      doc = win.document;
+      toggleExperiment(win, 'amp-render', true, true);
+    });
+
+    it('renders', async () => {
```

Comment:
Worth adding tests for amp-state data and remote fetched data, as well as unit tests for your helper fns.